src/Adapter/BaseAdapter.php

```diff
@@ -96,4 +103,25 @@ protected function getTimezoneOffset()
 
         return sprintf('%s%d:%02d', $prefix, $hours, $minutes);
     }
+
+    /**
+     * Validate the given select's order by rules
+     *
+     * @param Select $select
+     *
+     * @throws UnexpectedValueException In case the direction is invalid
+     */
+    protected function validateOrderBy(Select $select): void
```

Comment:
`ORDER BY` is not exclusive to `SELECT` statements, please use the appropriate interface.